### PR TITLE
Adds support for Postgres Databases

### PR DESCRIPTION
**Pull request by NobleMathews**
Implements task from https://github.com/deepsourcelabs/murdock/issues/21

Remove Elastic DB related logic
Adds support for Postgres test and production databases

---
Reimplemented due to git issues

### CODE REVIEW

File: cli.py [hunk 1]
```diff
@@ -1,33 +1,28 @@
...
 from inspector import make_multiple_requests
 
 app = typer.Typer(add_completion=False)
-configfile = configparser.ConfigParser()
 
 
 @app.callback(invoke_without_command=True)
 def main(
-    lang: str = typer.Option(...),
+    lang: str = typer.Option(None),
```
Comment (tushar-deepsource) [inline cli.py:21]: why is this set back to `None`? This is a required parameter

File: cli.py [hunk 2]
```diff
@@ -84,11 +62,11 @@ def main(
         payload[lang] = packages
     if lang not in ["go", "python", "javascript"]:
         raise LanguageNotSupportedError(lang)
-    if host and port:
-        es = connect_elasticsearch({"host": host, "port": port}, (es_uid, es_pass))
-    else:
-        logging.warning("Elastic not connected")
-        es = None
+    if psql := get_db():
+        if not db_name:
```
Comment (tushar-deepsource) [inline cli.py:66]: Why is db name optional anyway?

File: dependencies/helper.py [hunk 3]
```diff
@@ -126,7 +126,7 @@ def handle_npmjs(api_response: requests.Response, queries: dict, result: Result)
     dependencies_q: jmespath.parser.ParsedResult = queries["dependency"]
     repo_q: jmespath.parser.ParsedResult = queries["repo"]
     version = version_q.search(data)
-    if version is not None:
```
Comment (tushar-deepsource) [inline dependencies/helper.py:129]: why is this changed?

**Review by tushar-deepsource — CHANGES_REQUESTED** [review]

File: cli.py (hunk 2 above)
Comment (NobleMathews) [reply cli.py:66]: You can run the package without using the database, the initial plan was to keep the DB internal

File: cli.py (hunk 1 above)
Comment (NobleMathews) [reply cli.py]: Ah my bad

File: cli.py (hunk 2 above)
Comment (NobleMathews) [reply cli.py:66]: Sure i will look into it in another PR

**Comment by tushar-deepsource** [comment]
@NobleMathews Please add some example CLI commands along with the database configuration required to test this

**Review by tushar-deepsource — APPROVED** [review]

**Pull request merged**